### PR TITLE
Added Lightning related translations for Dutch

### PR DESCRIPTION
**Pull request by LinoxBE**

### CODE REVIEW

**Comment by LinoxBE** [comment]
Fixed the `Peer Info` into `Node Info`. And rolled back the wrong overwhite (stupid mistake). I removed in `Archived_Body` the dot at the end of the sentence, because the English version has no dot neither.

File: BTCPayServer/wwwroot/js/checkout/langs/nl.js [hunk 1]
```diff
@@ -44,8 +44,8 @@ Je kan terug komen naar {{storeName}} indien je nog eens je betaling wilt prober
     "This invoice has been paid": "Deze factuur werd betaald",
     // Invoice archived
     "This invoice has been archived": "Deze factuur werd geactiveerd",
-    "Archived_Body": "Please contact the store for order information or assistance",
+    "Archived_Body": "Bedankt om de winkel te contacteren voor bijstand met of informatie over deze bestelling",
 +    // Lightning
 +    "BOLT 11 Invoice": "BOLT 11 Factuur",
-+    "Peer Info": "Peer Info"
++    "Node Info": "Node Info"
```
Comment (lepipele) [inline BTCPayServer/wwwroot/js/checkout/langs/nl.js]: Just please make sure next time not to have + leading string if there is no need for that. Thanks!

Comment (LinoxBE) [reply BTCPayServer/wwwroot/js/checkout/langs/nl.js]: I didn't realise that. Thanks for your help again. It's clearly way too long ago that I worked with others on the same code!

Comment (lepipele) [reply BTCPayServer/wwwroot/js/checkout/langs/nl.js]: ;) glad I could help and thanks for your translations!

**Pull request merged**